v63003/u_inref/gtm8617.csh

```diff
@@ -0,0 +1,51 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Case for each functionality described in the release note
+#
+
+echo "# Null subscript set to never"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -N=never
```

Comment:
Please expand qualifier names in test scripts even though abbreviated forms are accepted. i.e. -N -> -NULL_SUBSCRIPTS, -std -> -stdnullcoll etc. in various places.

Reply:
The output of the MUPIP SET command is included in the reference file. That is not enough proof that null subscripts is set to never. One needs to do a **dse dump -file |& grep "Null subscripts"** to verify that is indeed set in the db file header. Same with the other commands/qualifiers in this test.